conbench/tests/api/test_runs.py

```diff
@@ -76,3 +77,14 @@ def test_run_list_filter_by_sha_and_machine_no_match(self, client):
         args = urllib.parse.urlencode(args)
         response = client.get(f"/api/runs/?{args}")
         self.assert_200_ok(response, [])
+
+
+class TestRunDelete(_asserts.DeleteEnforcer):
+    url = "api/runs/{}/"
+
+    def test_delete_run(self, client):
```

Comment:
Can you make this test follow the same pattern as:

```
    def test_delete_benchmark(self, client):
        self.authenticate(client)
        summary = _fixtures.create_benchmark_summary()

        # can get before delete
        Summary.one(id=summary.id)

        # delete
        response = client.delete(f"/api/benchmarks/{summary.id}/")
        self.assert_204_no_content(response)

        # cannot get after delete
        with pytest.raises(NotFound):
            Summary.one(id=summary.id)
```

Perhaps:

```
    def test_delete_benchmark(self, client):
        self.authenticate(client)
        summary = _fixtures.create_benchmark_summary()
        run_id = summary.run_id

        # can get before delete
        Run.one(id=run_id)

        # delete
        response = client.delete(f"/api/runs/{run_id}/")
        self.assert_204_no_content(response)

        # cannot get after delete
        with pytest.raises(NotFound):
            Run.one(id=run_id)
```

Reply:
Changed